org.sonarlint.eclipse.core.tests/src/test/java/org/sonarlint/eclipse/core/internal/jobs/AnalyzeStandaloneProjectJobTest.java

```diff
@@ -200,6 +204,46 @@ public void analyze_report_should_only_triggers_two_events_for_all_marker_operat
 
   }
 
+  @Test
+  public void analyzeWithQuickFixes() throws Exception {
+    IFile file = (IFile) project.findMember("src/main/java/com/quickfix/FileWithQuickFixes.java");
+    DefaultSonarLintProjectAdapter slProject = new DefaultSonarLintProjectAdapter(project);
+    IDocument document = new SimpleDocument("package com.quickfix;\n"
+      + "\n"
+      + "import java.util.*;\n"
+      + "\n"
+      + "public class FileWithQuickFixes {\n"
+      + "\n"
+      + "  public int foo(boolean a, String foo) {\n"
+      + "    List<String> list = new ArrayList<String>();\n"
+      + "    return 0;\n"
+      + "  }\n"
+      + "}");
```

Comment:
Do we have to repeat the file content here ? Couldn't we load it somehow ?

Reply:
I copied blindly what was done in other tests, but in fact it seems we are correctly loading content from disk.